deploy/sre-prometheus/100-sre-api-errors.PrometheusRule.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: sre-uptime-sla
+  namespace: openshift-monitoring
+spec:
+  groups:
+  - name: sre-uptime-sla
+    rules:
+    - alert: SLAUptimeSRE
+      annotations:
+        message: The API server has had 100 percent request failures for 300 seconds.
+      expr: |
+        (sum(rate(apiserver_request_count{job="apiserver", code=~"(400|5..)"}[5m])) / sum(rate(apiserver_request_count{job="apiserver"}[5m])) >= 1)
+      for: 5m
+      labels:
+        severity: warning
+        namespace: openshift-monitoring
```

Comment:
SRE will be alerted in PD because it's a warning and it's in a core namespace.  Based on the PR description it appears this is not desired, this is for reporting elsewhere.  Can we set the severity to something other than `warning` or `critical` and have it go to telemeter still?  If not, need to see this paired with a PR to configure-alertmanager-operator to suppress (route to `null`) this alert.

Reply:
Would it be possible to use `silent-test` alerts for this, considering they already exist?

Reply:
Thanks @jewzaam - Thanos requires either `warning` or `critical` so please find the paired PR for configure-alertmanager-operator here - https://github.com/openshift/configure-alertmanager-operator/pull/71; this dev/nulls the alert

Reply:
side node: warning/critical alerts, even if routed away in alert manager, will still show up in the cluster health view in OCM. I don't know if this is desired or not.